Library/AppEnvironment.swift

```diff
@@ -355,6 +355,14 @@ public struct AppEnvironment {
 
     userDefaults.set(data, forKey: environmentStorageKey)
   }
+
+  public static var appVersionString: String {
```

Comment:
Simply copy pasted here...but since it's on the actual `AppEnvironment` struct now it doesn't have to be `static`...would be happy to change.

Reply:
Sure lets make this a instance variable 👍